api/lib/api/clients/common.ex

```diff
@@ -0,0 +1,33 @@
+defmodule Tesla.Middleware.RequestLogger do
+  @behaviour Tesla.Middleware
+
+  def call(env, next, _) do
+    env
+    |> IO.inspect()
+    |> Tesla.run(next)
+  end
+end
+
+defmodule Tesla.Middleware.ReturnErrorTupleOnError do
+  @moduledoc """
+  Return :ok/:error tuples for successful HTTP transations, i.e. when the request is completed
+  (no network errors etc) - but it can still be an application-level error (i.e. 404 or 500)
+  """
+  def call(env, next, _opts) do
+    try do
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Prefer using an implicit `try` rather than explicit `try`.](https://app.codacy.com/manual/lucaskjaero/foreign-language-reader/pullRequest?prid=5112709)